### PR TITLE
test: move Postman collections to directory structure

### PR DESCRIPTION
**Pull request by knutties**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description
This change switches to using the `dir` version of newman where collections are maintained as directories instead of collection json files.  It uses the https://github.com/knutties/newman/tree/feature/newman-dir fork.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context
This change will help easier management and editing of postman tests using the engineer's favourite editor instead of being forced to use the Postman UI.

## How did you test it?
Ran postman tests on laptop via following script:

```
for i in postman/collection-dir/*   
do                  
    connector=$(basename $i);
    cargo run --bin test_utils -- --connector_name=$connector --base_url=https://integ-api.hyperswitch.io --admin_api_key=xxxxxx
done
```

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: .github/workflows/postman-collection-runner.yml [hunk 1]
```diff
@@ -107,6 +107,11 @@ jobs:
               fi
           done
 
+      - name: Install newman from fork
+        run: |
+          npm install -g 'git+ssh://git@github.com:knutties/newman.git#feature/newman-dir'
+          npm config get prefix
```
Comment (SanchithHegde) [inline .github/workflows/postman-collection-runner.yml:113]: Do we absolutely need the `npm config get prefix` here, or can we remove it?

Comment (knutties) [reply .github/workflows/postman-collection-runner.yml:113]: We can remove it - I kept it in to see what it outputs :)

Comment (knutties) [reply .github/workflows/postman-collection-runner.yml:113]: Screenshot 2023-08-24 at 8 08 01 AM

It is /usr/local

Comment (SanchithHegde) [reply .github/workflows/postman-collection-runner.yml]: > I kept it in to see what it outputs 

I figured that would be the case.

**Review by SanchithHegde — APPROVED** [review]

**Comment by pixincreate** [comment]
Updated branch to check if Sanity UI test would pass. Have raised another PR addressing the failure

**Review by ashokkjag — APPROVED** [review]

**Review by pixincreate — APPROVED** [review]

**Pull request merged**